### PR TITLE
kubernetes: Tags in Imagestream Create and Modify dialogs

### PR DESCRIPTION
**Pull request by stefwalter**
Add the missing Tags field.

### CODE REVIEW

**Comment by stefwalter** [comment]
screenshot from 2016-04-21 21-42-19

**Comment by stefwalter** [comment]
According to the designs here: https://raw.githubusercontent.com/cockpit-project/cockpit-design/master/kubernetes/images.png

**Comment by petervo** [comment]
tags

Padding doesn't look right on chrome.

Tags box doesn't expand when there are a lot in the screen shot the box in overflowing and extra tags are hidden. Happens with both chrome and FF

File: pkg/kubernetes/scripts/images.js [hunk 1]
```diff
@@ -575,25 +576,34 @@
...
-                populate: spec.dockerImageRepository ? "pull" : "none",
+                populate: populate,
                 pull: spec.dockerImageRepository || "",
+                tags: tagData.parseSpec(spec),
             };
 
             $scope.fields = fields;
             $scope.labels = {
                 populate: {
                     none: "Don't pull images automatically",
```
Comment (aweiteka) [inline pkg/kubernetes/scripts/images.js]: Let's label this "Create empty image stream"

File: pkg/kubernetes/scripts/images.js [hunk 2]
```diff
@@ -575,25 +576,34 @@
...
                     pull: "Pull all tags from another image repository",
```
Comment (aweiteka) [inline pkg/kubernetes/scripts/images.js]: This one is more accurate as "Sync all tags from a remote image repository"

**Comment by aweiteka** [comment]
Here are the user stories I think we want to target:

1. I want to import all image metadata from a remote repository. Example CLI cmd:

        oc import-image <imagestream_name> --all --confirm --from=<registry/repo/image> [--insecure]

1. I want to be able to enable remote registry tracking for an image. Example CLI cmd:

        oc tag --scheduled <registry/repo/image>:<tag> [--insecure]

1. I want to enable/disable/update remote tracking for one of the tags I discovered in step 1. Cmd is same as step 2.

File: pkg/kubernetes/scripts/images.js (hunk 1 above)
Comment (stefwalter) [inline pkg/kubernetes/scripts/images.js]: Good idea. Made this happen when creating a dialog.

File: pkg/kubernetes/scripts/images.js (hunk 2 above)
Comment (stefwalter) [inline pkg/kubernetes/scripts/images.js]: Done.

**Comment by stefwalter** [comment]
> Here are the user stories I think we want to target:

These are good for follow up pull requests right?

**Comment by stefwalter** [comment]
> Tags box doesn't expand when there are a lot in the screen shot the box in overflowing and extra tags are hidden. Happens with both chrome and FF

Fixed.

**Comment by aweiteka** [comment]
> These are good for follow up pull requests right?

Yes. And maybe we might need a bit of design on this. A checkbox for insecure? Particularly challenging is how we would visualize the "scheduled" or "watch" boolean per tag.

**Pull request closed without merging**